packages/sit-onyx/src/styles/required.scss

```diff
@@ -16,7 +16,12 @@
 
 .onyx-optional-marker {
   .onyx-use-optional &::after {
-    font-family: var(--onyx-font-family);
+    font: {
+      family: var(--onyx-font-family);
+      size: 1rem;
```

Comment:
Shouldn't the font size be inherited? So that it aligns if we use a different density